src/main/java/io/whelk/asciidoc/TemplateMojo.java

```diff
@@ -45,20 +43,47 @@
     @Parameter(defaultValue = "${project}", required = true, readonly = true)
     MavenProject project;
 
+    // @VisibleForTesting
+    Map<String, String> vars = Map.of();
+
     @SneakyThrows
     public void execute() throws MojoExecutionException, MojoFailureException {
         setDefaultConfiguration();
 
         final var lines = this.readLines(templateDirectory, templateFile);
+        this.vars = loadVars(lines);
         final var updatedLines = this.updateLines(lines);
 
         Files.write(Paths.get(outputDirectory, outputFile), updatedLines);
     }
 
     @SneakyThrows
     private List<String> readLines(String first, String... more) {
-        return new ArrayList<>(Files
+        ArrayList<String> strings = new ArrayList<>(Files
                 .readAllLines(Paths.get(first, more)));
+        return strings;
+    }
+
+    // @VisibleForTesting
```

Comment:
Can we remove this, if it is not needed?

Reply:
It's just to indicate why the relaxed access level. Want me to change it to a comment or leave it out?
Guava has an annotation like that but it has no functional use. Just for reading. 
I was accidentally using the testing libraries one, but I'd prefer to keep the test library scope as test.

Reply:
Ah ok, apologies for the delay in responding. That should be fine.